src/pages/Admin/Queue/Manage/Manage.js

```diff
@@ -118,6 +196,72 @@ function Manage({ queueId, openSnackbar }) {
       });
   };
 
+  const handleAddCounter = () => {
+    //TODO validate max capacity, if it capped, an error message should appear.
+    if (counter >= maxCapacity) {
+      setAlertOpen(true);
+      setCounter(prevState => prevState + 1);
+    } else {
+      setCounter(prevState => prevState + 1);
+    }
+  };
+
+  const handleRemoveCounter = () => {
+    if (counter > 0) {
+      setCounter(prevState => prevState - 1);
+    }
+  };
```

Comment:
as a suggestion, return first...something like this:
```suggestion
    if (counter <= 0) return:

     setCounter(prevState => prevState - 1);
    
  };
```
why? if you need to add more logic, you don't need to nest it in the if....it stays more clear ;)